rules/ask-for-help/rule.md

```diff
@@ -4,14 +4,33 @@ title: Do you know when to ask for help?
 uri: ask-for-help
 authors:
   - title: Sam Wagner
+  - title: Piers Sinclair
+    url: https://ssw.com.au/people/piers-sinclair
 related:
   - do-you-use-timeboxing-to-avoid-wasted-time
+  - do-you-manage-up
+  - do-you-add-context-reasoning-to-your-emails
 created: 2022-08-30T04:06:01.553Z
 guid: 113b1e11-6df4-4901-9e9f-bcc06b6fb352
 ---
 When you're working, you will encounter issues that block you from progressing, force you to do a lot of research or otherwise spend time building towards being able to complete your work. In these moments, the question arises, "When should I ask for help?"
 
```

Comment:
missing line 

"This concept is critical when trying to talk with Senior developers because their time is much more valuable. If you [call them without having done any research, it will waste a lot of their time and yours](https://www.ssw.com.au/rules/investigate-your-question-for-two-minutes-before-asking-someone-on-im) and won't be a good [ROI](https://www.ssw.com.au/rules/return-on-investment)"

Reply:
Added :)